packages/chains/chains/1422.json

```diff
@@ -11,8 +11,8 @@
   },
   "infoURL": "https://polygon.technology/solutions/polygon-zkevm/",
   "shortName": "zkevmtest",
-  "chainId": 1402,
-  "networkId": 1402,
+  "chainId": 1422,
+  "networkId": 1422,
```

Comment:
This get automatically pulled from the ethereum-list/chains repo, so it'll get overwritten. Best would be to put up a PR on their repo of it's wrong. Otherwise overwrite it in the script

Reply:
gotcha

Reply:
https://github.com/DefiLlama/chainlist/ is it this repo?

Reply:
https://GitHub.com/ethereum-list/chains

Reply:
ah okay

Reply:
Easy enough to override in our script that downloads all the chains tho, we already have an override for localhost in there

Reply:
hmm so should we just override instead?